README.md

```diff
@@ -1 +1,82 @@
-# Mobile-portfolio-setup
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Table of Contents
+
+  * About the Project
+   - Built With
+
+  *Getting Started
+    - Prerequisites
+    - Setup
+    - Install
+    - Usage
+    - Run test
+    - Deployment
+
+  * Authors
+
+  * Contributing
+
+  * Show your support
+
+  * Acknowledgements
+
+  * License
+
+# PORTFOLIO-SETUP
+
+> My first project at microverse with a simple message "Hello Microverse"
```

Comment:
I suggest adding a description that is specific to this project in order to maintain professional documentation.